alphalens/performance.py

```diff
@@ -453,9 +453,11 @@ def average_cumulative_return_by_quantile(quantized_factor, forward_returns,
     ----------
     quantized_factor : pd.Series
         Factor quantiles indexed by date and asset.
-    forward_returns : pd.Series - MultiIndex
-        Daily forward returns indexed by date and asset and
-        optional a custom group.
+    prices : pd.DataFrame
```

Comment:
So, I made a bad choice the first time I wrote average_cumulative_return_by_quantile and I wanted to fix it.